hub/core/chunk_engine/write.py

```diff
@@ -89,49 +90,37 @@ def write_bytes(
             `IndexMeta.add_entry` supports more parameters than this. Anything passed in this dict will also be used
             to call `IndexMeta.add_entry`.
     """
+    # TODO pass CHUNK_MIN, CHUNK_MAX and read from tensor_meta instead of using constants
```

Comment:
reminder

you should update `hub.htypes`

Reply:
specifically: https://github.com/activeloopai/Hub/blob/2e6fdda73a38cf904aa8af7c11bde0d8c2203fc7/hub/htypes.py#L16

Reply:
Resolving for now. Added to the backlog as discussed.